src/core/core.datasetController.js

```diff
@@ -594,7 +598,7 @@ export default class DatasetController {
 
     for (i = 0, ilen = parsed.length; i < ilen; ++i) {
       value = parsed[i][scale.axis];
-      if (!isNaN(value)) {
+      if (value !== null && !isNaN(value)) {
```

Comment:
Use `isFinite` for consistency and perhaps performance?

Reply:
done